aio/aio-proxy/aio_proxy/parameters.py

```diff
@@ -143,9 +149,37 @@ def parse_and_validate_terms(request) -> str:
         )
 
 
-def extract_parameters(
+def parse_and_validate_latitude(request):
+    try:
```

Comment:
A bit weird to duplicate error message. I guess you could just `raise ValueError` as message will get overwritten by line 158